### DISCUSSION TITLE
Who's using/谁在使用 Loggie?

### DISCUSSION THREAD

**Discussion by ethfoo** [loggie-io/loggie#407] [General]
<img src="https://user-images.githubusercontent.com/3616466/201591997-c89d04e5-be80-497d-9c05-1b156a745340.svg" width="50%">

> Many hands make light work.
> 众人拾柴火焰高

The Loggie community can only grow with the combined efforts of every user and developer.
Loggie 社区的发展离不开每一位用户和开发者的共同努力。

The Loggie community appreciates every user and developer who uses and cares about Loggie. We are committed to listening to each one of you to make the community more prosperous.
感激每一位使用和关注 Loggie 的用户和开发者，我们努力倾听每一个人的声音，让社区变得更加繁荣。

We look forward to your comments in the comments space, including but not limited to the following areas.
我们期待您能在评论区留言，其内容包括但不限于以下方面。

- Organization / 组织
- Region / 地区
- Contact / 联系方式
- Usage Scenario / 使用场景、使用方式或者使用了哪些组件均可
- Desired Features: features you care about but don't implement(optional) / 期望功能：关心的但暂时未实现（可选）

**Comment by ethfoo** [comment]
<img src="https://user-images.githubusercontent.com/3616466/201593594-a046aaa3-201f-44da-a819-9bd4988802ff.svg" width="40%">

- Organization / 组织:  [NetEase DigitalSail](sf.163.com).
- Region / 地区:  Hangzhou, China
- Contact / 联系方式:  loggie_io@163.com
- Usage Scenario / 使用场景:  We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, prometheusExporter, unix), interceptor(transformer, logAlert), sink(elasticsearch, kafka, file)

**Comment by LeiZhang-Hunter** [comment]
<img src="https://eoitek.com/images/eoi_logo.png" width="40%">

- Organization / 组织:  [Eoitek](https://eoitek.com/).
- Region / 地区:  Shanghai, China
- Contact / 联系方式: [ info@eoitek.com](https://eoitek.com/)
- Usage Scenario / 使用场景:  We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, unix), interceptor(transformer), sink(elasticsearch, kafka, file)

**Comment by zanderme** [comment]
<img src="https://user-images.githubusercontent.com/9931786/201596306-76cd838a-c1b0-4e28-b2fc-176fc0771ad7.png" width="40%">

- Organization / 组织:  [明源云客电子商务有限公司](https://www.myunke.com).
- Region / 地区:  ShenZhen, China
- Contact / 联系方式:  400@mingyuanyun.com
- Usage Scenario / 使用场景:  We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, prometheusExporter), interceptor(transformer), sink(kafka, file)

**Comment by techtqmall** [comment]
<img src="https://user-images.githubusercontent.com/22021017/201603949-6bafbbc4-6707-4ad1-a411-fe6b641f425f.png" width="40%">

Organization / 组织: [杭州艾麦科技有限公司](https://www.imile.com/).
Region / 地区: Hangzhou, China
Contact / 联系方式: [ali-admin@imile.me)
Usage Scenario / 使用场景: We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, prometheusExporter), interceptor(transformer), sink(kafka, file)

**Comment by machine3** [comment]
![image](https://user-images.githubusercontent.com/74903810/202099143-0e2fbef3-b4a9-4988-af9f-99d71d4d9ba7.png)
- Organization / 组织:  [网易传媒](https://163.com).
- Region / 地区:  Beijing, China
- Contact / 联系方式:  wangmc7@163.com
- Usage Scenario / 使用场景:  We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, prometheusExporter, unix), interceptor(transformer, logAlert), sink(elasticsearch, kafka, file)

**Comment by xiaojiayu404** [comment]
![image](https://user-images.githubusercontent.com/13671286/215233514-cff68f1a-de2d-488a-b5fc-0dde840ca751.png)

- Organization / 组织: [芒果TV](https://www.mgtv.com/)
- Region / 地区: Changsha, China
- Contact / 联系方式: (mailto: hitedu2015@163.com)
- Usage Scenario / 使用场景: We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, prometheusExporter, unix), interceptor(transformer, logAlert), sink(elasticsearch, kafka, file)

**Comment by Steelki** [comment]
![rmzk-logo](https://user-images.githubusercontent.com/34326414/216928598-77bce3df-a868-495e-b846-d55939080fe2.svg)

- Organization / 组织: [人民中科](https://www.people-ai.cn/)
- Region / 地区: Changsha, China
- Contact / 联系方式: (mailto: [17600250792@163.com]）
- Usage Scenario / 使用场景: We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, prometheusExporter, unix), interceptor(transformer, logAlert), sink(elasticsearch, kafka, file)

**Comment by flyhighzy** [comment]
<img width="300" alt="logo" src="https://user-images.githubusercontent.com/1246062/221772712-62f39532-019a-45d1-ac62-cbce9bca806c.png">

Organization / 组织: [上海人工智能实验室](https://www.shlab.org.cn/)
Region / 地区: Shanghai, China
Contact / 联系方式: (mailto: [comm@pjlab.org.cn](mailto:comm@pjlab.org.cn)）
Usage Scenario / 使用场景: We use Loggie as the engine of the log platform for log collection, forwarding and processing under Kubernetes. Components used include: source(file, k8s pod), interceptor(transformer, multiline..), sink(elasticsearch, loki)
